Client/Application/LeanplumIntegration.swift

```diff
@@ -127,35 +127,24 @@ class LeanPlumClient {
     private var prefs: Prefs? { return profile?.prefs }
     private var enabled: Bool = true
     private var setupType: LPSetupType = .none
-    var didReceiveLPStartResponse: Bool = false
+    // Boolean variable to indicate whether leanplum has finished starting-up
+    var startCallFinished: Bool = false
+    // Closure delegate for when leanplum has finished starting-up
+    var finishedStartingLeanplum: (() -> Void)?
     // leanplumDeviceId is what comes directly from leanplum and
     // should be used and kept for showing the device ID in the
     // debug menu so we can easily find the user in LP dashboard
     var leanplumDeviceId: String? {
         return Leanplum.deviceId()
     }
-    // Closure delegate for when leanplum start finishes
-    var onStartLPVariable: ((LPVar?) -> Void)?
-    // This is only added for testing purpose for issue #6656.
-    // In general we get device ID from IDFV and #6656
-    var phoneDeviceId: String {
-        set(value) {
-                UserDefaults.standard.set(value, forKey: LPDeviceIDKey)
-        }
-        get {
-            return UserDefaults.standard.string(forKey: LPDeviceIDKey) ?? ""
-        }
-    }
-    let onboardingABTestVariable = LPVariables.showOnboardingScreenAB
+    // Used for sentry logging to know about the current state of leanplum execution
     var lpState: LPState = .disabled
     // This defines an external Leanplum varible to enable/disable FxA prepush dialogs.
     // The primary result is having a feature flag controlled by Leanplum, and falling back
     // to prompting with native push permissions.
     private var useFxAPrePush = LPVar.define("useFxAPrePush", with: false)
```

Comment:
nit: legacy code we should remove at some point. Same with the line below it I assume.

Reply:
This another one of LP's things if I get rid of any variables defined in code it gets rid of them from their server side as soon as someone runs debug version of leanplum integration 🤦 

Will likely just move it to our LPVariable struct to unclutter the integration class